packages/storebuilder/src/wizards/store-setup/screens/StoreDetails.tsx

```diff
@@ -3,19 +3,17 @@ import { WizardSectionTitle, FormField, TextInput, Form } from '@stellarwp/wme-u
 import { Stack } from '@mui/material';
 import ScreenWrapper from '@setup/ScreenWrapper';
 import { useStoreSetup } from '@store/hooks';
-import { FtcFormItemsInterface } from '@setup/data/store-setup-screen-data';
+// import { FtcFormItemsInterface } from '@setup/data/store-setup-screen-data';
```

Comment:
couple commented out lines here...should we just delete these? Or do you want to come back to them?

Reply:
I'll take care of those puppies as I'm work through the Store Setup wizard.